installer/bundle/bundle_skel.sh

```diff
@@ -61,6 +61,8 @@ UNSUPPORTED_OPENSSL=60
 INSTALL_PYTHON_CTYPES=61
 INSTALL_TAR=62
 INSTALL_SED=63
+INSTALL_CURL=64
+INSTALL_GPG=65
```

Comment:
I am not sure why it is required to duplicate these docs in two different GitHub repos instead of having them linked? I will see if I can have that also fixed today. Most likely a separate PR since it belongs to extension repro.